tests/test_core.py

```diff
@@ -192,6 +192,26 @@ def test_check_max_window_size():
             core.check_window_size(m, max_size=3)
 
 
+def test_check_window_size_excl_zone_case1():
```

Comment:
This test is redundant. The ` test_check_window_size_excl_zone_case2` should suffice.